src/Hintman/Suggestion/Aggregation.hs

```diff
@@ -0,0 +1,28 @@
+module Hintman.Suggestion.Aggregation
+       ( aggregate
+       ) where
+
+import Hintman.Suggestion.Core (Line (..), Suggestion (..), toLines)
+
+import qualified Hintman.Suggestion.TrailingNewline as STN
+import qualified Hintman.Suggestion.TrailingSpaces as STS
+
+
+aggregate :: FilePath -> Text -> [Suggestion]
+aggregate path txt = concat [deleteSuggest, editSuggest]
```

Comment:
we're not dealing with different suggestions for the same line for now? /cc @ChShersh

Reply:
@sphaso Yes, not now. Will solve the problem under different issue.